formatters/basic/config.go

```diff
@@ -34,7 +34,8 @@ func DefaultConfig() *Config {
 		lineBreakStyle = yamlfmt.LineBreakStyleCRLF
 	}
 	return &Config{
-		Indent:     2,
-		LineEnding: lineBreakStyle,
+		Indent:           2,
+		LineEnding:       lineBreakStyle,
+		RetainLineBreaks: true,
```

Comment:
My rule has been that if there's a feature that has to hack around a shortcoming of the yaml library, I have it turned off by default to make the most stable possible default experience (despite weird behaviours from the library). 

Does the vscode extension not read from the working directory's `.yamlfmt` or accept options from `settings.json`?

Reply:
OK, it's fine. I suppose everybody needs the feature and I could make it default in vscode.